src/TransitionSystems/location_id.rs

```diff
@@ -21,13 +23,52 @@ impl LocationID {
             "consistency: {}",
             string
         ))
+        .unwrap()
         .remove(0);
 
         match query {
             QueryExpression::Consistency(x) => (*x).into(),
             _ => unreachable!(),
         }
     }
+
+    /// This function is used when you want to compare [`LocationID::Simple`] location to [`LocationID::AnyLocation`].
```

Comment:
I understand what this means, but the functions is used when you want to compare with a partial locationID (which then ofcourse contains LocationID::AnyLocation)

Reply:
fixed